api_testing/test_getting_resource.py

```diff
@@ -0,0 +1,12 @@
+from schemas.data_preparation import ValidationJsonSchemas
+
+
+def test_get_user_1(session, base_url):
+    get_user_1 = session.get(base_url + '1')
+    assert get_user_1.status_code == 200
+    ValidationJsonSchemas.checking_json_schema(url=get_user_1)
+
+
+def test_get_invalid_data(session, base_url):
+    get_invalid_data = session.get(base_url + '0')
+    assert get_invalid_data.status_code == 404
```

Comment:
You could add **-1** and **201** todos ids to complete boundary value analysis. Also you could add check length of API response dictionary:

```
@pytest.mark.parametrize('todo_id', [-1, 0, 201])
def test_get_invalid_data(session, base_url, todo_id):
    res = session.get(base_url + str(todo_id))
    assert res.status_code == 404
    assert len(res.json()) == 0
```